### ISSUE TITLE
Implement support for checkpoint models (e.g. "MDX23C-8KFFT-InstVoc_HQ_2.ckpt")

### ISSUE THREAD

**Issue by ajayarora1235** [karaokenerds/python-audio-separator#42]
This should be a pretty simple fix -- the bug comes from there not being a "primary_stem" label for this model in the config json. Perhaps remove this requirement or have a default value for it?

**Comment by beveradb** [comment]
Not sure exactly what you mean here, I don't think implementing support for the checkpoint models is as trivial as you think 😅 

There's a whole separate class for the inference process in UVR, so it'll need a chunk of time for me to figure out and refactor into this codebase.
If nobody else raises a PR implementing it, I'll get around to it eventually though!

**Comment by ajayarora1235** [comment]
ahh I see. I thought you had already implemented .ckpt support, but I now realize it was just .pth. Apologies!

**Comment by zhzhongshi** [comment]
hi @beveradb ，
I found a repo that would work with that model `MDX23C-8KFFT-InstVoc_HQ.ckpt`
 [ultimatevocalremover_api](https://github.com/NextAudioGen/ultimatevocalremover_api)

 clone it to local
cd ultimatevocalremover_api
pip install -r requirememts.txt

put the `MDX23C-8KFFT-InstVoc_HQ.ckpt` into `src\models_dir\mdxc\weights\MDX23C-8KFFT-InstVoc_HQ`

new file test.py
```python
from src import models
import torch
import audiofile

def test_mdxc_load():
    if torch.cuda.is_available(): device = "cuda"
    elif torch.backends.mps.is_available(): device = torch.device("mps")
    else: device = "cpu"
    print("device:", device)
    mdxc = models.MDXC(name="MDX23C-8KFFT-InstVoc_HQ", other_metadata={}, device=device, logger=None)
    
    name = "1.wav"
    # Separating an audio file
    seperted_audio = mdxc(name)
    assert seperted_audio is not None
    stems= seperted_audio["separated"]
    print("stems:", stems.keys())
    vocals= stems["vocals"]
    instrumental = stems["instrumental"]
    vocals_path = "vocals_mdxc.wav"
    instrumental_path = "instrumental_mdxc.wav"
    print("output vocals:",vocals_path)
    audiofile.write(vocals_path, vocals, mdxc.sample_rate)
    print("output instrumental:",instrumental_path)
    audiofile.write(instrumental_path, instrumental, mdxc.sample_rate)

if __name__ == "__main__":
    test_mdxc_load()
```
run python test.py and it works :)

**Comment by beveradb** [comment]
Hey @ajayarora1235 , just a heads up, audio-separator version `0.16.2` or higher now supports the checkpoint models, including the VIP models like `MDX23C-8KFFT-InstVoc_HQ_2.ckpt` 👀

Huge thanks to @zhzhongshi for getting the ball rolling with the MDXC implementation 🙇 

If you do choose to use one of the VIP models though, please consider subscribing to Anjok's Patreon if you aren't already: 
https://www.patreon.com/uvr

Hope this helps! 🎉
